pkg/envoy/lds/listener.go

```diff
@@ -200,3 +150,124 @@ func getCIDRRange(cidr string) (*xds_core.CidrRange, error) {
 
 	return cidrRange, nil
 }
+
+// getFilterForService builds a network filter action for traffic destined to a specific service
+func getFilterForService(dstSvc service.MeshService, cfg configurator.Configurator) (*xds_listener.Filter, error) {
```

Comment:
To make it clear these are outbound filters, I'd rename this and other similar functions related to outbound filter processing.

```suggestion
func getOutboundFilterForService(dstSvc service.MeshService, cfg configurator.Configurator) (*xds_listener.Filter, error) {
```